packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2107,7 +2107,7 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
                     },
                     internalId: {
                         value: internalId,
-                        tag: TelemetryDataTag.PackageData,
```

Comment:
I do not think this change can be done right now. applications need to ensure they handle TelemetryDataTag.CodeArtifact properly. There was no mention of TelemetryDataTag.CodeArtifact in breaking.md up till now, so apps do not know much about it, and likely do not have it in their switch statements.

The issue with starting to use it without giving earlier hands up is that different layers are delivered at different speeds to customers. So new container runtime (using this new tag) will be running in older host apps that do not handle this tag yet.

Reply:
I noted this deprecation in BREAKING.md for 1.0.0 and mentioned to move to the new `CodeArtifact` tag. What's the correct process here?

Reply:
Sorry, I did not see it there - my mistake